### PR TITLE
Insectoid Fashion Month 1 - Masks

### PR DESCRIPTION
**Pull request by PlausibleDeniability**
There isn't enough insect variants to clothing, here's the majority of missing masks.  These are either re-positioned and tweaked human sprites or redone to fit the insectoid face

Balaclava | Neorussian Mask
--- | ---
balaclava  | nr_mask

SWAT/Syndicate | HECU
--- | ---
swat | hecu 

Deathsquad| Chem mask
--- | ---
death | chemmask

Red Bandana | Up
--- | ---
bandred | bandred_up

Blue Scarf | Up
--- | ---
blue_scarf | blue_scarf_up

Green Scarf | Up
--- | ---
green_scarf | green_scarf_up

Red Scarf | Up
--- | ---
red_scarf | red_scarf_up

Chapmask | Joy | Happiest
--- | --- | ---
chapmask | joy | happiest

Psychadelic Clown Mask | Sexy Clown Mask | Wizard Clown Mask
--- | --- | ---
clownmaskpsyche | sexyclown |  wizzclown 

Lola | Disguise Moustache| Golden Mask
--- | --- | ---
lola | fake-moustache | goldenmask

Luchador | Tecnicos | Rudos
--- | --- | ---
luchador | luchag | luchar

Monkey Mask | Owl Mask | Pig Mask
--- | --- | ---
monkeymask | owl | pig

Oni Mask | Plague Doctor | Xeno Necklace
--- | --- | ---
onimask | plaguedoctor | xeno_necklace

[sprites]
[qol]
[tested]

:cl:
 * imageadd: Created insectoid variants for the majority of leftover masks
 * tweak: Insectoids have gained the ability to contort their heads to fit in balaclavas and other masks!

### CODE REVIEW

**Comment by PrimeDSS13** [comment]
image
snug bug

also weird color to showcase this stuff but I won't judge

**Comment by PlausibleDeniability** [comment]
> 
> 
> image
> snug bug
> 
> also weird color to showcase this stuff but I won't judge

These be Terry colours
Photoshop_l04HbTWMDh

**Comment by ShiftyRail** [comment]
What a beautiful IoC (Insectoid of Colour)
I hate it

**Comment by beesil** [comment]
very nice spritework

I hate to be the 'whatabout' guy but if you're looking for more to do, antique gasmasks still use the human sprites on bugs.

**Comment by PlausibleDeniability** [comment]
> 
> 
> very nice spritework
> 
> I hate to be the 'whatabout' guy but if you're looking for more to do, antique gasmasks still use the human sprites on bugs.

Yeah good catch, I've included it as part of the pull request, should have included it in the screenshots but I just copied one of the existing gas mask sprites

```DM
/obj/item/clothing/mask/gas/ert
	name = "antique gas mask"
	desc = "A face-covering mask that can be connected to an air supply."
	icon_state = "ert"
	siemens_coefficient = 0.7
	species_fit = list(VOX_SHAPED, INSECT_SHAPED)
	w_class = W_CLASS_SMALL
	can_flip = 0
	canstage = 0
```

**Comment by SonixApache** [comment]
>scarf support for bugs

b-but the scarves don't even have sprites for humans

File: code/modules/clothing/masks/miscellaneous.dm [hunk 1]
```diff
@@ -78,8 +81,8 @@
 	flags = FPRINT
 	body_parts_covered = HIDEHAIR | MOUTH
 	w_class = W_CLASS_SMALL
-	species_fit = list(VOX_SHAPED, GREY_SHAPED)
-	species_restricted = list("exclude", INSECT_SHAPED)
+	species_fit = list(VOX_SHAPED, GREY_SHAPED, INSECT_SHAPED)
+	// species_restricted = list("exclude", INSECT_SHAPED)
```
Comment (DamianX) [inline code/modules/clothing/masks/miscellaneous.dm]: delete this instead of commenting it out

**Review by DamianX — COMMENTED** [review]
it's good other than that

File: code/modules/clothing/masks/miscellaneous.dm (hunk 1 above)
Comment (PlausibleDeniability) [reply code/modules/clothing/masks/miscellaneous.dm]: Done

**Pull request merged**